cmd/abapEnvironmentCheckoutBranch.go

```diff
@@ -74,38 +75,44 @@ func runAbapEnvironmentCheckoutBranch(options *abapEnvironmentCheckoutBranchOpti
 	client.SetOptions(clientOptions)
 	pollIntervall := com.GetPollIntervall()
 
-	log.Entry().Infof("Starting to switch branch to branch '%v' on repository '%v'", options.BranchName, options.RepositoryName)
-	log.Entry().Info("--------------------------------")
-	log.Entry().Info("Start checkout branch: " + options.BranchName)
-	log.Entry().Info("--------------------------------")
+	repositories := []abaputils.Repository{}
+	err = checkCheckoutBranchRepositoryConfiguration(*options)
```

Comment:
IMO this would introduce one more not needed if clause that wouldn't make thinks more readable and rather more complex.
According to your suggestion the pattern would be:
```
err = checkCheckoutBranchRepositoryConfiguration(*options)
if err != nil {
  return err 
}

err = someOtherLogic()

if err != nil {
  return err 
}
```

whereas the current implementation makes things more readable since complexity is not that great.

```
err = checkCheckoutBranchRepositoryConfiguration(*options)
if err == nil {
  err = someOtherLogic()
}
if err != nil {
  return err 
}
```